pkgs/top-level/release.nix

```diff
@@ -11,7 +11,7 @@
 { nixpkgs ? { outPath = (import ../../lib).cleanSource ../..; revCount = 1234; shortRev = "abcdef"; revision = "0000000000000000000000000000000000000000"; }
 , officialRelease ? false
   # The platforms for which we build Nixpkgs.
-, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" ]
+, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ]
```

Comment:
I'm curious what adding `aarch64-darwin` here means. Going by what's in `release.nix`, it's going to add a bunch of jobs to Hydra that will be required to progress channels. (But is that per-platform or do all platforms need to succeed?)

Reply:
Oh, I think #208888 answers my question.